guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -2,14 +2,18 @@
 
 = Configuring Repositories
 
-Use this procedure to enable the repositories that are required to install {ProductName}. Choose from the available list which operating system and version you are installing on:
-
+Use this procedure to enable the repositories that are required to install {ProductName}.
+ifndef::satellite[]
+Choose from the available list which operating system and version you are installing on:
+endif::[]
```

Comment:
```suggestion
endif::[]

```
Add a blank line, otherwise the list doesn't render correctly on non-sat builds.

Reply:
Addressed :+1: